pkg/scheduler/plugins/rescheduling/low_node_utilization.go

```diff
@@ -202,3 +203,13 @@ func isContinueEvictPods(usage *NodeUtilization, totalAllocatableResource map[v1
 	}
 	return true
 }
+
+// resourceExceedsThrehold judges if resource percent exceeds the threhold.
+func resourceExceedsThrehold(reqPercent float64, rName v1.ResourceName, config interface{}) bool {
```

Comment:
Thanks for the optimization. It makes sense to deal with evicted pods with more cosideration such as resource request, priority and any other factors. I think it may be more reasonable to list all the conditions which should be give more notice and deal with them in a framework. So do you have some ideas or suggestions about that?